kde-misc/plasma-applet-window-buttons/plasma-applet-window-buttons-9999.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN/plasma-/}"
+
+KFMIN=5.38.0
+QTMIN=5.9.0
+PLASMA_MINIMAL=5.12.0
```

Comment:
You can do that; But please note that KDE Frameworks 5.37 was dropped more than two years ago, and Qt 5.7 was gone soon afterwards (5.8 never packaged), and Plasma 5.11 as well. So you are quite safe from a packaging standpoint, not least because `ecm.eclass` already sets `KFMIN=5.64.0` anyway. Unlike gnome proj we are not overzealous with minimum versions. ;)

Regarding `PLASMA_MINIMAL`: a variable used only once should rather not be declared at all.